personality_insights/ApplicationBuilder/personality-insights-text-widget.erb

```diff
@@ -0,0 +1,60 @@
+<%
+
+  text_to_analyze = @subject['title'].join(" ")
+model = endpoint("personality_insights_text", text: text_to_analyze)
```

Comment:
We should have a comment in the code here about modifying this line to send the text to the proxy, if WEX < 11.